modules/react/popup/stories/Popup.stories.mdx

```diff
@@ -0,0 +1,483 @@
+import {Meta} from '@storybook/addon-docs/blocks';
+
+import {Popup} from '@workday/canvas-kit-react/popup';
+import {Specifications} from '@workday/canvas-kit-docs';
+
+import {Simple} from './examples/Simple';
+import {InitialFocus} from './examples/InitialFocus';
+import {OpenWithoutModel} from './examples/OpenWithoutModel';
+import {ManualControl} from './examples/ManualControl';
+import {MultiplePopups} from './examples/MultiplePopups';
+import {NestedPopups} from './examples/NestedPopups';
+import {FocusRedirect} from './examples/FocusRedirect';
+import {FocusTrap} from './examples/FocusTrap';
+import {RTL} from './examples/RTL';
+import {PopupModelConfigComponent, PopupStateComponent, PopupEventsComponent} from './PopupModel';
+
+<Meta title="Components/Popups/Popup/React" component={Popup} />
+
+# Canvas Kit Popups
+
+A "popup" is a classification for a type of stacked UI element that appears "on top" of statically
+positioned content. Tooltips, Modals, Dropdown menus, etc are all examples of "popups". Canvas Kit
+has a "stack manager" system for managing these popups. Different types of popups have different
+requirements of behavior for UX and accessibility - we can call them behaviors, capabilities, or
+traits. Canvas Kit comes with a number of [behaviors](#hooks) in the form of React Hooks.
+
+You should use the most semantic component for your use-case before using `Popup` directly, like
+`Modal`, which already has the correct behaviors built-in. If no component already exists, you can
+use `Popup` and use our [hooks](#hooks). The `Popup` component comes with a `Popup.Popper` component
+that positions a popup using [PopperJS](https://popper.js.org/) that registers a popup with the
+`PopupStack` automatically and sets the popup model's `placement` property. `Popup.Popper` component
+and hooks work with the stack management system for correct rendering and accessibility behavior. If
+you cannot use `Popup.Popper`, use the [usePopupStack](#usepoupstack) hook to properly register and
+deregister the popup at the correct time. If you cannot use our hooks, consider upgrading your
+component to use Hooks. If you cannot do that, you'll have to look up the `PopupStack` package for
+the direct API and have a look at the source code for our hooks into the `PopupStack` API.
+
+This package comes with everything you need to build Popup UIs.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+## Usage
+
+The `Popup` component is a low-level compound component that is used to build popup UIs that are not
+already covered by Canvas Kit.
+
+### Basic
```

Comment:
Nit: The story is called `Simple`, but the heading is `Basic`. Let's align on one.